package.json

```diff
@@ -41,5 +41,13 @@
     "prettier": "prettier",
     "tsc": "tsc",
     "multiapi": "ts-node ./scripts/multiapi.ts"
+  },
```

Comment:
Please remove the changes to this file and the package-lock.json

Reply:
Hi @weshaggard , remove the changes from the package file and package-lock.json file.